WORKSPACE.bazel

```diff
@@ -26,7 +27,17 @@ load("@rules_nodejs//nodejs:repositories.bzl", "nodejs_register_toolchains")
 
 nodejs_register_toolchains(
     name = "nodejs",
-    node_version = "16.17.1",
+    # The below can be removed once @rules_nodejs/nodejs is updated to latest which contains https://github.com/bazelbuild/rules_nodejs/pull/3701
```

Comment:
Isn't there a release for this?

Reply:
I tried dropping the `node_repositories`, but it failed for find the Node version so I assume it hasn't been released/it doesn't have the exact version we need.